_scripts/instruction-widget/templates/getting-started/webroot.html

```diff
@@ -0,0 +1,35 @@
+<p>
+Certbot supports a number of different “plugins” that can be used to obtain and/or install certificates.
+</p>
+<p>
+Since your server architecture doesn't yet {{officially}} support automatic
+installation {{imperative}} use use the certonly command to obtain a cert.
+</p>
+<p>
+To obtain a cert using the <a href="https://certbot.eff.org/docs/using.html#webroot">webroot</a> plugin, which can work with the webroot directory of any webserver software, run this command.
+</p>
+
+{{#xenial}}
+<aside class="note">
+    <h4>Note:</h4>
+    <p>Ubuntu Xenial has an old version of Certbot packaged for it that lacks a few features. For example the certonly option isn't available, and the <a href="https://certbot.eff.org/docs/using.html#webroot">webroot plugin</a> doesn't have an interactive mode. You can still get a certificate but you'll need to specify all of your variables on the command line.
```

Comment:
> For example the certonly option isn't available

`certonly` is available on Xenial.

> You can still get a certificate but you'll need to specify all of your variables on the command line.

Unless there is another feature we're missing in Xenial you'd like to mention, what if we just combined this and the previous sentence to say something like:

"The webroot plugin doesn't have an interactive mode, so you'll need to specify your domains and webroot paths on the command line to use the plugin."